### PR TITLE
UAF-4458 Bug - Copied DV documents can be submitted with inactivated Payment Reason

### PR DESCRIPTION
**Pull request by adamkost**
Created edu.arizona DisbursementVoucherPaymentReasonValidation.java
Updated bean id="DisbursementVoucher-paymentReasonValidation" to reflect the new validation class
plus supporting constants and error message.

### CODE REVIEW

File: kfs-core/src/main/resources/edu/arizona/kfs/fp/document/validation/configuration/DisbursementVoucherValidation.xml [hunk 1]
```diff
@@ -11,6 +11,12 @@
                            http://www.springframework.org/schema/aop
```
Comment (ghost) [inline kfs-core/src/main/resources/edu/arizona/kfs/fp/document/validation/configuration/DisbursementVoucherValidation.xml]: This override belongs in edu/arizona/kfs/fp/document/validation/configuration/FinancialProcessingValidators.xml. The edu/arizona/kfs/fp/document/validation/configuration/DisbursementVoucherValidation.xml file should only contain beans related to document events, addAccountingLineValidation, updateAccountingLineValidation, routeDocumentValidation. etc.

**Review by ghost — COMMENTED** [review]
Initial review complete. Please see my comment.

File: kfs-core/src/main/resources/edu/arizona/kfs/fp/document/validation/configuration/DisbursementVoucherValidation.xml (hunk 1 above)
Comment (adamkost) [reply kfs-core/src/main/resources/edu/arizona/kfs/fp/document/validation/configuration/DisbursementVoucherValidation.xml]: Changed.

**Comment by ghost** [comment]
Looks good!

**Pull request merged**